api/RS.API/RS.Service/Logic/OpeningManagerService.cs

```diff
@@ -167,14 +165,10 @@ public IResult UpdateOpening(OpeningViewModel openingViewModel)
 
                 if (openingSkillList.Any())
                 {
-                    _openingRepository.UpdateOpeningSkills(openingSkillList);
-                }
-                else
-                {
-                    _openingRepository.Update(openingModel);
-                    _openingRepository.SaveChanges();
-
+                    _openingRepository.UpdateOpeningSkills(openingSkillList);  
                 }
+                _openingRepository.Update(openingModel);
+                _openingRepository.SaveChanges();
```

Comment:
why save changes here in service layer ??

Reply:
Done